info.bibletime.BibleTime.json

```
@@ -0,0 +1,80 @@
+{
+    "id": "info.bibletime.BibleTime",
+    "branch": "stable",
+    "sdk": "org.kde.Sdk",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "command": "bibletime",
+    "tags": ["stable"],
+    "modules": [
+	{
+	    "name": "clucene",
+	    "buildsystem": "cmake-ninja",
+	    "config-opts" : [
+		"-DCMAKE_BUILD_TYPE=release"
+	    ],
+	    "builddir": true,
+	    "sources": [ { 
+		    "type": "git", 
+		    "url": "git://clucene.git.sourceforge.net/gitroot/clucene/clucene", 
+		    "commit": "e8e3d20f20da5ee3e37d347207b01890829a5475" 
+		} 
+	    ],
+            "cleanup": [
+                "/lib/pkgconfig",
+                "/lib/CLuceneConfig.cmake",
+                "/lib/debug",
+                "/include"
+            ]
+	},
+	{
+	    "name" : "sword",
+	    "sources" : [
+		{
+		  "type" : "archive",
+                  "url": "http://ci.crosswire.org/job/sword-release/18/artifact/sword-1.8.1.tar.gz",
+                  "sha256": "ce9aa8f721a737f406115d35ff438bd07c829fce1605f0d6dcdabc4318bc5e93"
+		}
+	    ],
+            "cleanup": [
+                "/lib/pkgconfig",
+                "/lib/libsword.la",
+                "/lib/debug",
+                "/include"
+	    ]
+	},
+        {
+            "name": "bibletime",
+            "buildsystem": "cmake-ninja",
+	    "config-opts" : [
+		"-DCMAKE_BUILD_TYPE=release"
+	    ],
+            "builddir": true,
+            "sources": [ { 
+
+                    "type": "archive", 
+                    "url": "https://github.com/bibletime/bibletime/releases/download/v2.11.2/bibletime-2.11.2.tar.xz",
+                    "sha256": "d178197ce87c8211733409fb28fa6ed0220800bbf06944ebb8fe85bf78ddabe8"
+                },
+                {
+                    "type": "patch", 
+                    "path": "bibletime.patch"
+                }
+	    ],
+            "cleanup": [
+                "/lib/pkgconfig",
+                "/lib/debug"
+	    ]
+        }
+    ],
+    "finish-args": [
+        "--socket=x11",
```

```suggestion
        "--share=ipc",
```
If you use the x11 socket you need ipc share as well. per: http://docs.flatpak.org/en/latest/sandbox-permissions.html